design/common.blocks/link/_theme/link_theme_islands.styl

```diff
@@ -18,6 +18,21 @@
         {
             color: #070;
         }
+
+        &_ghost
+        {
+            color: #999;
+        }
+
+        &_black
+        {
+            color: #000;
+        }
+
+        &_strong
```

Comment:
I believe such things must be solved with proper design guides that based on strict and clear colour schemes, not by weird CSS class names, like `p_color_blue`.

Apart from discussions, current implementation doesn't allow to compose `_external` and `_strong` links, while this combination is used in the company you've been talking about ;)